core/src/Kokkos_ExecPolicy.hpp

```diff
@@ -220,11 +220,11 @@ class RangePolicy
    }
 
    member_type new_chunk_size = 1;
-   while(new_chunk_size*100*concurrency < m_end-m_begin)
```

Comment:
this is something we need to check. I am not sure that the heuristic to go this low is good on most platforms.

Reply:
Sure, that's understandable. I'm happy to remove this if it's fairly likely that I'll be able to customize this for the HPX backend at a later point.